src/createApp.ts

```diff
@@ -512,6 +513,20 @@ function addIdentifier(app: App, id: Identifier) {
 	};
 }
 
+function createCustomWidget(app: App, id: string) {
+	const customFactories = customElementFactories.get(app);
+
+	return app.defaultWidgetStore.get(id).then((state: any) => {
```

Comment:
ah, I had assumed it worked more like a Map/Set which simply returned `undefined` versus a throw.  If we add the negative tests and the results are "logical" in how they behave.